sycl/include/CL/sycl/properties/accessor_properties.hpp

```diff
@@ -17,19 +17,28 @@ __SYCL_INLINE_NAMESPACE(cl) {
 namespace sycl {
 namespace property {
 
-class noinit : public detail::DataLessProperty<detail::NoInit> {};
+class no_init : public detail::DataLessProperty<detail::NoInit> {};
+
+class __SYCL2020_DEPRECATED("spelling is now: no_init") noinit
+    : public detail::DataLessProperty<detail::NoInit> {};
 
 } // namespace property
 
 #if __cplusplus > 201402L
 
-inline constexpr property::noinit noinit;
+inline constexpr property::no_init no_init;
+
+inline constexpr property::noinit
+    noinit __SYCL2020_DEPRECATED("spelling is now: no_init");
```

Comment:
```
D:\BuildBot\Product_worker_intel\sycl-win-x64-pr\llvm.obj\.\bin\..\include\sycl\CL/sycl/properties/accessor_properties.hpp(32,11): error: expected ';' after top level declarator
    noinit __SYCL2020_DEPRECATED("spelling is now: no_init");
```
MSVC can't compile this declaration.

Reply:
I repositioned the macro call and I expect MSVC will be happy with that. I also now use the `__SYCL_INLINE_CONSTEXPR` macro there.